BrickController2/BrickController2.WinUI/PlatformServices/GameController/GameControllerService.cs

```diff
@@ -54,19 +53,19 @@ public event EventHandler<GameControllerEventArgs> GameControllerEvent
         }
     }
 
-    internal void RaiseEvent(IDictionary<(GameControllerEventType, string), float> events)
+    internal void RaiseEvent(IDictionary<(GameControllerEventType, string), float> events, string controllerDeviceId)
     {
         if (!events.Any())
         {
             return;
         }
 
-        GameControllerEventInternal?.Invoke(this, new GameControllerEventArgs(events));
+        GameControllerEventInternal?.Invoke(this, new GameControllerEventArgs(controllerDeviceId, events));
     }
 
-    internal void RaiseEvent(string deviceId, string key, GameControllerEventType eventType, float value = 0.0f)
+    internal void RaiseEvent(string deviceId, string key, GameControllerEventType eventType, string controllerDeviceId, float value = 0.0f)
```

Comment:
This method is not used.

Reply:
As muc as I know, this method is from István... I can delete it if you want?